src/Api/Application.php

```diff
@@ -100,6 +111,7 @@ protected function getDefaultCommands(): array
         $commands[] = new VersionCommand();
         $commands[] = new GetConfigCommand();
         $commands[] = new SetConfigCommand();
+        $commands[] = new DebugAccesskeyCommand();
```

Comment:
Why are you not passing `$this->projectDir` in the constructor?

Reply:
Changed in 0b49f9a3a1b0f5c9ba27b01644c82bd551dfe76d (for all commands actually)